src/lib/Scenes/City/City.tsx

```diff
@@ -103,16 +110,17 @@ export class CityView extends Component<Props, State> {
   }
 
   componentDidUpdate() {
-    if (!this.props.isDrawerOpen && this.scrollView) {
-      this.scrollView.scrollTo({ x: 0, y: 0, animated: true })
-    }
-
     if (this.state.buckets) {
       // We have the Relay response; post a notification so that the ARMapContainerViewController can finalize the native UI.
       NativeModules.ARNotificationsManager.postNotificationName("ARLocalDiscoveryQueryResponseReceived", {})
     }
   }
 
+  setSelectedTab(selectedTab) {
+    this.setState({ selectedTab: selectedTab.i }, this.fireScreenViewAnalytics)
```

Comment:
@orta Do you recall why this was needed? No other code in this component seems to use it, but calling `setState` does lead to an extra render pass of this component.

Reply:
I took it directly from the homescreen - not sure if it's needed

Reply:
Gotcha, thanks!